test/test_nn.py

```diff
@@ -6519,18 +6519,32 @@ def test_noncontig_conv_grad_cuda(self, dtype=torch.float):
         self.assertEqual(result, input.grad.data, atol=dtype2prec_DONTUSE[dtype], rtol=0)
 
     def test_pixel_shuffle(self):
```

Comment:
This test looks good; is there also a test for the two error cases? If not, let's add one to verify the module throws both those exceptions as expected.